eras/shelley-ma/test-suite/src/Test/Cardano/Ledger/Mary/Golden.hs

```diff
@@ -84,107 +84,116 @@ goldenScaledMinDeposit =
     [ testCase "one policy, one (smallest) name" $
         scaledMinDeposit
           ( MaryValue 1407406 $
-              Map.singleton pid1 (Map.fromList [(smallestName, 1)])
+              MultiAsset $
+                Map.singleton pid1 (Map.fromList [(smallestName, 1)])
           )
           minUTxO
           @?= Coin 1407406,
       testCase "one policy, one (small) name" $
         scaledMinDeposit
           ( MaryValue 1444443 $
-              Map.singleton
-                pid1
-                (Map.fromList [(smallName 1, 1)])
+              MultiAsset $
+                Map.singleton
+                  pid1
+                  (Map.fromList [(smallName 1, 1)])
```

Comment:
Couldn't  we use  valueFromList here? (and elsewhere)
valueFromList 1444443 [(pid1,smallName 1,1)]